serializations.md

```diff
@@ -482,9 +504,8 @@ The channel offchain transaction is not included directly in the transaction tre
 
 #### Channel
 ```
-[ <id>               :: binary()
```

Comment:
Not sure whether [off-chain channel state update](https://github.com/aeternity/protocol/blob/6614f009a3d033e3f8aec6a381d16def766f7eaa/epoch/api/channels_api_usage.md#channel-state) requires update. This would also depend on how epoch code changes contextual with this PR are performed (e.g. usage of PoI in same PR? cc @velzevur ).

Reply:
I believe that the actual hashes in the intended usage shouldn't be used anyway. They depend on the public keys etc. If a reader decodes the example hashes and finds that the protocol has changed, this is probably fine. The structure of the messages remain the same.